src/pybind/mgr/dashboard/frontend/src/app/shared/services/notification.service.ts

```diff
@@ -82,4 +85,15 @@ export class NotificationService {
         break;
     }
   }
+
+  notify(finishedTask: FinishedTask, success: boolean = true) {
```

Comment:
Could you rename the method to be easier to identify, something like `notifyTask`.

Reply:
Done